Lumpen Radio/Radio.swift

```diff
@@ -56,32 +60,87 @@ class Radio: ObservableObject {
     // Reference:
     // https://stackoverflow.com/questions/48555049/how-to-stream-audio-from-url-without-downloading-mp3-file-on-device
     func startRadio() {
+        if self.player != nil {
+            self.player?.play()
+            self.isPlaying = true
+            self.bottomText = BOTTOM_TEXT_RADIO_ON
+            return
+        }
         guard let url = URL(string: LUMPEN_AUDIO_URL) else {
             print("Could not start radio")
             return
         }
-        let playerItem = AVPlayerItem(url: url)
+        self.playerItem = AVPlayerItem(url: url)
         let player = AVPlayer(playerItem: playerItem)
-        let audioSession = AVAudioSession.sharedInstance()
+        
+        player.play()
+        self.isPlaying = true
+        self.bottomText = BOTTOM_TEXT_RADIO_ON
+        self.player = player    }
+    
+    func stopRadio() {
+        self.player?.pause()
+        // no need to release the player, as it will cause a delay when tuning in again
+//        self.player = nil
```

Comment:
Sounds good, let's at least delete the `self.player = nil` comment and leave the other line as documentation

Reply:
Done